lisa/tools/mount.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from pathlib import PurePosixPath
+from typing import Any, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.util import LisaException
+
+
+class Mount(Tool):
+    __UMOUNT_ERROR_PATTERN = re.compile(r".*mountpoint not found", re.MULTILINE)
+
+    @property
+    def command(self) -> str:
+        return self._command
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def operations(
+        self, disk: str, point: str, command: str = "mount", erase: bool = True
```

Comment:
It can be two methods, `mount` and `umount`. It's not a good practice to pass the command in. The logic is different, and the str may have typo, and others don't know what's the right value.